nilearn/mass_univariate/tests/test_permuted_least_squares.py

```diff
@@ -595,19 +595,27 @@ def test_tfce_smoke(random_state=0):
 
 def test_cluster_level_parameters_smoke(random_state=0):
     """Test combinations of parameters related to cluster-level inference."""
+    import random
     import nibabel as nib
     from nilearn.maskers import NiftiMasker
 
+    random.seed(random_state)
+    np.random.seed(random_state)
```

Comment:
```suggestion
    rng = np.random.RandomState(0)
```

Reply:
Oh cool! I did not know about this and that the Python stdlib module “random” is aware of this.

Q: Shouldn't this state be inherited from the function input `random_state` instead of hard coding it to 0?

Reply:
@jrdalenberg yes my mistake,  it should take `random_state` as an argument